source/tutorials/using-api-interface.rst

```diff
@@ -1,144 +1,23 @@
-represent all thermometers that will be used in dojot. This template would have
-only one attribute called, let's say, "temperature". While creating the device,
-the user would select its "physical template", let's say *TexasInstr882*, and
-the 'thermometer' template. The user would have also to add translation
-instructions (implemented in terms of data flows, build in flowbuilder) in
-order to map the temperature reading that will be sent from the device to a
-"temperature" attribute.
-
-In order to create a device, a user selects which templates are going to
-compose this new device. All their attributes are merged together and
-associated to it - they are tightly linked to the original template so that any
-template update will reflect all associated devices.
-
-The component responsible for managing devices (both real and virtual) and
-templates is `DeviceManager`_. `DeviceManager documentation`_ explains in more
-depth all the available operations.
-
-
-Flows
-*****
-
-A flow is a sequence of blocks that process a particular event or device
-message. It contains:
-
-- entry point: a block representing what is the trigger to start a particular
-  flow;
-- processing blocks: a set of blocks that perform operations using the event.
-  These blocks may or may not use the contents of such event to further process
-  it. The operations might be: testing content for particular values or ranges,
-  geo-positioning analysis, changing message attributes, perform operations on
-  external elements, and so on.
-- exit point: a block representing where the resulting data should be forwarded
-  to. This block might be a database, a virtual device, an external element,
-  and so on.
-
-The component responsible for dealing with such flows is `flowbroker`_.
-
-Step-by-step device management
-******************************
+Using API interface
+===================
 
 This section provides a complete step-by-step tutorial of how to create,
-update, send messages to and check historical data of a device. We will create
-a simple device with only one attribute, send a few messages emulating the
-physical device and check the historical data for the only attribute this
-device has.
+update, send messages to and check historical data of a device. Also, this
+tutorial assumes that you are using `docker-compose`_, which has all the
+necessary components to properly run dojot.
+
+.. note::
+   - Audience: developers
+   - Level: basic
+   - Reading time: X m
 
-Also, this tutorial assumes that you are using `docker-compose`_, which has all
-the necessary components to properly run dojot (so all API requests will be
-sent to localhost:8000).
 
 Getting access token
-++++++++++++++++++++
+--------------------
 
-As said in `User authentication`_, all requests must contain a valid access
+As said in :ref:`User authentication`, all requests must contain a valid access
 token. You can generate a new token by sending the following request:
 
 .. code-block:: bash
```

Comment:
On the following excerpt from this section:
"All further requests will use a bash variable called bash ${JWT}, which contains the token got from auth component.".
 I think it does not sound good this "bash", it seems to be unnecessary.

Reply:
Ok